src/Magento/FunctionalTestingFramework/Console/RunManifestCommand.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types = 1);
+
+namespace Magento\FunctionalTestingFramework\Console;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+class RunManifestCommand extends Command
+{
+    /**
+     * The return code. Determined by all tests that run.
+     *
+     * @var integer
+     */
+    private $returnCode = 0;
+
+    /**
+     * A list of tests that failed.
+     * Eg: "tests/functional/tests/MFTF/_generated/default/AdminLoginTestCest.php:AdminLoginTest"
+     *
+     * @var string[]
+     */
+    private $failedTests = [];
+
+    /**
+     * Configure the run:manifest command.
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $this->setName("run:manifest")
+            ->setDescription("runs a manifest file")
+            ->addArgument("path", InputArgument::REQUIRED, "path to a manifest file");
+    }
+
+    /**
+     * Executes the run:manifest command.
+     *
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     * @return integer
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $manifestFile = file($input->getArgument("path"), FILE_IGNORE_NEW_LINES);
```

Comment:
if path/to/your/testManifest.txt is invalid it throws a failed to open steam exception, how about adding an explicit error checking,
`if (!realpath($input->getArgument("path")))`

Reply:
I added an explicit check for the input file.